api/clients/v2/payloaddispersal/payload_disperser.go

```diff
@@ -73,14 +70,23 @@ func (pd *PayloadDisperser) SendPayload(
 	// payload is the raw data to be stored on eigenDA
 	payload *coretypes.Payload,
 ) (*coretypes.EigenDACert, error) {
+	return pd.SendPayloadWithProbe(ctx, payload, nil)
+}
+
+// SendPayloadWithProbe is the same as SendPayload, but allows the caller to pass in a probe for collecting metrics.
+// If the probe is nil then no metrics will be collected.
+func (pd *PayloadDisperser) SendPayloadWithProbe(
```

Comment:
Here's the spot we discussed offline- recomment sticking with the single `SendPayload` method strategy, to be compatible with a future `PayloadDisperserMetric` struct

Reply:
I've removed the probe as a function parameter.